diff -r 92c9748e0ccb src/os_unix.c
--- a/src/os_unix.c	Sun Oct 06 17:46:56 2013 +0200
+++ b/src/os_unix.c	Sun Oct 27 12:33:20 2013 -0200
@@ -827,7 +827,7 @@
 		|| MAC_OS_X_VERSION_MAX_ALLOWED <= 1040)
 	/* missing prototype.  Adding it to osdef?.h.in doesn't work, because
 	 * "struct sigaltstack" needs to be declared. */
-	extern int sigaltstack __ARGS((const struct sigaltstack *ss, struct sigaltstack *oss));
+	extern int sigaltstack __ARGS((const stack_t *restrict ss, stack_t *restrict oss));
 #  endif

 #  ifdef HAVE_SS_BASE
